api/v1alpha4/types.go

```diff
@@ -706,7 +706,8 @@ type Volume struct {
 
 	// Encrypted is whether the volume should be encrypted or not.
 	// +optional
-	Encrypted bool `json:"encrypted,omitempty"`
+	// +kubebuilder:default=true
```

Comment:
```suggestion
	// +kubebuilder:default=true
```

We still don't want any defaulting here. Remove this.